Spatial.gd

```diff
@@ -857,6 +883,9 @@ func refresh_trail2d_points(trail2d: Line2D):
 ################################################################################
 # Signal Functions
 ################################################################################
+func _on_SaveTrail_pressed():
+	save_map_data(self.map_id)
+
```

Comment:
Where is this called? It seems to be new to this PR but nothing in this PR calls it.

Reply:
I don't see it in the negative diff, I think it is being created in this PR. I think if it is not being used then it should be removed form this PR and added to whatever PR is actually going to use it.

Reply:
Oh sorry I took it as this commit. Thats understandable, I've remove for now